script.module.enum34/addon.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.enum34" version="1.1.6" name="enum" provider-name="jmooremcc">
+    <requires>
+        <import addon="xbmc.python" version="2.24.0" />
+    </requires>
+    <extension library="lib/enum" point="xbmc.python.module" />
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">A Python interface for enum34</summary>
+        <description lang="en_GB">A Python interface for enum34</description>
+        <forum></forum>
```

Comment:
remove

Reply:
I used the addon.xml example from @Lunatixz for my addon.xml file.
I changed the the appropriate parts of the example file as needed.

Reply:
remove

Reply:
Just for clarification, are you asking me to remove the entire extension point or just a portion of it?

Reply:
this comment is just about the forum tag